src/main/java/kitchenpos/products/tobe/domain/ProductPrice.kt

```diff
@@ -0,0 +1,15 @@
+package kitchenpos.products.tobe.domain
+
+import jakarta.persistence.Column
+import jakarta.persistence.Embeddable
+import java.math.BigDecimal
+
+@Embeddable
+data class ProductPrice(
+    @Column(name = "price", nullable = false)
+    val value: BigDecimal
```

Comment:
data class 대신에 [Inline Class(= Value Class)](https://devs0n.tistory.com/83)를 활용해볼 수 있을 것 같아요. 🙂

```suggestion
@JvmInline
value class ProductPrice(
    val value: BigDecimal
```

Reply:
value 라는 키워드는 처음 봤는데 진짜 유용하네요.
jpa 의존성도 없애고 Embedded, Embeddable 어노테이션도 필요가 없어졋네요 
wrapping하는데에 참 좋은것같습니다 좋은 의견감사합니다~